packages/box/box.ts

```diff
@@ -116,7 +116,7 @@ const Box = {
     const tmpDir = tmp.dirSync({ unsafeCleanup });
     const unboxOptions = { logger, force };
     await Box.unbox(
-      `https://github.com/trufflesuite/truffle-init-${name}`,
+      `https://github.com:trufflesuite/truffle-init-${name}`,
```

Comment:
This syntax seems wrong to me - http git URLs don't use `:` the way that `git@github.com` URLs do. Is there a technical reason for this?

Reply:
This library wants this format and will not respond to the proper URL format above. You can check out examples [here](https://www.npmjs.com/package/download-git-repo#examples).